layout-server/src/main/java/org/finos/vuu/layoutserver/controller/LayoutController.java

```diff
@@ -52,28 +43,29 @@ public LayoutResponseDTO getLayout(@PathVariable UUID id) {
      */
     @ResponseStatus(HttpStatus.OK)
     @GetMapping("/metadata")
-    public List<MetadataResponseDTO> getMetadata() {
+    public List<MetadataResponseDto> getMetadata() {
 
-        return metadataService.getMetadata()
+        return layoutService.getMetadata()
             .stream()
-            .map(metadata -> mapper.map(metadata, MetadataResponseDTO.class))
+            .map(metadata -> mapper.map(metadata, MetadataResponseDto.class))
             .collect(java.util.stream.Collectors.toList());
     }
 
     /**
      * Creates a new layout
      *
      * @param layoutToCreate the layout to be created
-     * @return the layout that has been created, with the autogenerated ID and created date
+     * @return the layout that has been created
      */
     @ResponseStatus(HttpStatus.CREATED)
     @PostMapping
-    public LayoutResponseDTO createLayout(@Valid @RequestBody LayoutRequestDTO layoutToCreate) {
+    public LayoutResponseDto createLayout(
+        @RequestBody @Valid LayoutRequestDto layoutToCreate) {
```

Comment:
This can go back to how it was before, except DTO

Reply:
is it just the new line and the order of the annotations?

Reply:
discussed offline: undo new line

Reply:
done